calitp/pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "calitp"
+version = "2023.2.10"
+description = "Shared code for the Cal-ITP data codebases"
+authors = ["Andrew Vaccaro <atvaccaro@gmail.com>"]
```

Comment:
Please don't use a personal email here

Reply:
Ah sure thing, poetry pulls it from git and I've never thought about it. I will adjust elsewhere, too.